### ISSUE TITLE
DeepLabCut/Anipose conda env. MacOS [feature request and errors]?

### ISSUE THREAD

**Issue by MMathisLab** [lambdaloop/anipose#16]
**Is your feature request related to a problem? Please describe.**

on MacOS, Catalina 
conda env with deeplabcut and calligator installed as follows:

DLC-calligator.yaml 
```
# DLC-calligator.yaml 
#
# install: conda env create -f DLC-calligator.yaml 
# update:  conda env update -f DLC-calligator.yaml 

name: DLC-calligator
dependencies:
  - python=3.7
  - pip
  - jupyter
  - nb_conda
  - tensorflow==1.13.1
  - wxpython
  - Shapely
  - pip:
    - deeplabcut
    - calligator
```

Error:
When trying to install anipose with pip inside this env, there are issues with installing `lapsolver`:

```
mwmathis@Mackenzies-MacBook-Pro Desktop % pip install anipose
Collecting anipose
  Using cached anipose-0.6.6-py3-none-any.whl (54 kB)
Collecting scikit-video
  Using cached scikit_video-1.1.11-py2.py3-none-any.whl (2.3 MB)
Requirement already satisfied: calligator>=0.3.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (0.3.2)
Requirement already satisfied: opencv-python in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (3.4.9.31)
Requirement already satisfied: scipy in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (1.4.1)
Requirement already satisfied: tqdm in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (4.43.0)
Requirement already satisfied: pandas in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (1.0.1)
Requirement already satisfied: click in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (7.0)
Requirement already satisfied: toml in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (0.10.0)
Requirement already satisfied: numpy in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (1.16.4)
Requirement already satisfied: deeplabcut>=2.0.4.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from anipose) (2.1.6.2)
Collecting lapsolver>=1.0.2
  Using cached lapsolver-1.1.0.tar.gz (261 kB)
Requirement already satisfied: pillow in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from scikit-video->anipose) (7.0.0)
Requirement already satisfied: numba in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from calligator>=0.3.2->anipose) (0.48.0)
Requirement already satisfied: opencv-contrib-python~=3.4 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from calligator>=0.3.2->anipose) (3.4.9.31)
Requirement already satisfied: python-dateutil>=2.6.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from pandas->anipose) (2.8.1)
Requirement already satisfied: pytz>=2017.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from pandas->anipose) (2019.3)
Requirement already satisfied: scikit-learn in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.22.2.post1)
Requirement already satisfied: tensorpack>=0.9.7.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.9.9)
Requirement already satisfied: six in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (1.14.0)
Requirement already satisfied: setuptools in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (45.2.0.post20200210)
Requirement already satisfied: chardet in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (3.0.4)
Requirement already satisfied: ipython in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (7.12.0)
Requirement already satisfied: wheel in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.34.2)
Requirement already satisfied: ruamel.yaml~=0.15 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.16.10)
Requirement already satisfied: easydict in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (1.9)
Requirement already satisfied: moviepy in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (1.0.1)
Requirement already satisfied: h5py~=2.7 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (2.10.0)
Requirement already satisfied: tables in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (3.6.1)
Requirement already satisfied: scikit-image in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.16.2)
Requirement already satisfied: patsy in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.5.1)
Requirement already satisfied: statsmodels in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.11.1)
Requirement already satisfied: certifi in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (2019.11.28)
Requirement already satisfied: imgaug in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.4.0)
Requirement already satisfied: pyyaml>=5.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (5.3)
Requirement already satisfied: requests in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (2.23.0)
Requirement already satisfied: matplotlib==3.0.3 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (3.0.3)
Requirement already satisfied: ipython-genutils in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (0.2.0)
Requirement already satisfied: intel-openmp in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from deeplabcut>=2.0.4.1->anipose) (2019.0)
Requirement already satisfied: llvmlite<0.32.0,>=0.31.0dev0 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from numba->calligator>=0.3.2->anipose) (0.31.0)
Requirement already satisfied: joblib>=0.11 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from scikit-learn->deeplabcut>=2.0.4.1->anipose) (0.14.1)
Requirement already satisfied: tabulate>=0.7.7 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tensorpack>=0.9.7.1->deeplabcut>=2.0.4.1->anipose) (0.8.6)
Requirement already satisfied: psutil>=5 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tensorpack>=0.9.7.1->deeplabcut>=2.0.4.1->anipose) (5.7.0)
Requirement already satisfied: msgpack>=0.5.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tensorpack>=0.9.7.1->deeplabcut>=2.0.4.1->anipose) (1.0.0)
Requirement already satisfied: pyzmq>=16 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tensorpack>=0.9.7.1->deeplabcut>=2.0.4.1->anipose) (18.1.1)
Requirement already satisfied: termcolor>=1.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tensorpack>=0.9.7.1->deeplabcut>=2.0.4.1->anipose) (1.1.0)
Requirement already satisfied: msgpack-numpy>=0.4.4.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tensorpack>=0.9.7.1->deeplabcut>=2.0.4.1->anipose) (0.4.4.3)
Requirement already satisfied: traitlets>=4.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (4.3.3)
Requirement already satisfied: prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (3.0.3)
Requirement already satisfied: jedi>=0.10 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (0.16.0)
Requirement already satisfied: backcall in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (0.1.0)
Requirement already satisfied: pickleshare in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (0.7.5)
Requirement already satisfied: appnope; sys_platform == "darwin" in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (0.1.0)
Requirement already satisfied: decorator in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (4.4.1)
Requirement already satisfied: pexpect; sys_platform != "win32" in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (4.8.0)
Requirement already satisfied: pygments in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ipython->deeplabcut>=2.0.4.1->anipose) (2.5.2)
Requirement already satisfied: ruamel.yaml.clib>=0.1.2; platform_python_implementation == "CPython" and python_version < "3.9" in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from ruamel.yaml~=0.15->deeplabcut>=2.0.4.1->anipose) (0.2.0)
Requirement already satisfied: proglog<=1.0.0 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from moviepy->deeplabcut>=2.0.4.1->anipose) (0.1.9)
Requirement already satisfied: imageio<3.0,>=2.5; python_version >= "3.4" in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from moviepy->deeplabcut>=2.0.4.1->anipose) (2.8.0)
Requirement already satisfied: imageio-ffmpeg>=0.2.0; python_version >= "3.4" in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from moviepy->deeplabcut>=2.0.4.1->anipose) (0.4.1)
Requirement already satisfied: numexpr>=2.6.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from tables->deeplabcut>=2.0.4.1->anipose) (2.7.1)
Requirement already satisfied: networkx>=2.0 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from scikit-image->deeplabcut>=2.0.4.1->anipose) (2.4)
Requirement already satisfied: PyWavelets>=0.4.0 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from scikit-image->deeplabcut>=2.0.4.1->anipose) (1.1.1)
Requirement already satisfied: Shapely in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from imgaug->deeplabcut>=2.0.4.1->anipose) (1.6.4.post2)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from requests->deeplabcut>=2.0.4.1->anipose) (1.25.8)
Requirement already satisfied: idna<3,>=2.5 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from requests->deeplabcut>=2.0.4.1->anipose) (2.9)
Requirement already satisfied: kiwisolver>=1.0.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from matplotlib==3.0.3->deeplabcut>=2.0.4.1->anipose) (1.1.0)
Requirement already satisfied: cycler>=0.10 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from matplotlib==3.0.3->deeplabcut>=2.0.4.1->anipose) (0.10.0)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from matplotlib==3.0.3->deeplabcut>=2.0.4.1->anipose) (2.4.6)
Requirement already satisfied: wcwidth in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0->ipython->deeplabcut>=2.0.4.1->anipose) (0.1.8)
Requirement already satisfied: parso>=0.5.2 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from jedi>=0.10->ipython->deeplabcut>=2.0.4.1->anipose) (0.6.1)
Requirement already satisfied: ptyprocess>=0.5 in /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages (from pexpect; sys_platform != "win32"->ipython->deeplabcut>=2.0.4.1->anipose) (0.6.0)
Building wheels for collected packages: lapsolver
  Building wheel for lapsolver (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py'"'"'; __file__='"'"'/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-wheel-2gzfjn4b
       cwd: /private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/
  Complete output (77 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.9-x86_64-3.7
  creating build/lib.macosx-10.9-x86_64-3.7/lapsolver
  copying lapsolver/__init__.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver
  creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
  copying lapsolver/tests/__init__.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
  copying lapsolver/tests/test_files.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
  copying lapsolver/tests/test_dense.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
  running egg_info
  writing lapsolver.egg-info/PKG-INFO
  writing dependency_links to lapsolver.egg-info/dependency_links.txt
  writing top-level names to lapsolver.egg-info/top_level.txt
  reading manifest file 'lapsolver.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  writing manifest file 'lapsolver.egg-info/SOURCES.txt'
  creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/data
  creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs0.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs1.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs10.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs2.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs3.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs4.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs5.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs6.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs7.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs8.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  copying lapsolver/data/dense/costs9.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
  creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/etc
  copying lapsolver/etc/benchmark-dtype-int.png -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/etc
  copying lapsolver/etc/benchmark-dtype-numpy.float32.png -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/etc
  running build_ext
  Traceback (most recent call last):
    File "/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py", line 22, in run
      out = subprocess.check_output(['cmake', '--version'])
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 411, in check_output
      **kwargs).stdout
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 488, in run
      with Popen(*popenargs, **kwargs) as process:
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 800, in __init__
      restore_signals, start_new_session)
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 1551, in _execute_child
      raise child_exception_type(errno_num, err_msg, err_filename)
  FileNotFoundError: [Errno 2] No such file or directory: 'cmake': 'cmake'
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py", line 88, in <module>
      keywords='hungarian munkres kuhn linear-sum-assignment bipartite-graph lap'
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages/setuptools/__init__.py", line 144, in setup
      return distutils.core.setup(**attrs)
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 966, in run_commands
      self.run_command(cmd)
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages/wheel/bdist_wheel.py", line 223, in run
      self.run_command('build')
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/command/build.py", line 135, in run
      self.run_command(cmd_name)
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py", line 25, in run
      ", ".join(e.name for e in self.extensions))
  RuntimeError: CMake must be installed to build the following extensions: lapsolverc
  ----------------------------------------
  ERROR: Failed building wheel for lapsolver
  Running setup.py clean for lapsolver
Failed to build lapsolver
Installing collected packages: scikit-video, lapsolver, anipose
    Running setup.py install for lapsolver ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py'"'"'; __file__='"'"'/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-record-ppsvuzrf/install-record.txt --single-version-externally-managed --compile --install-headers /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/include/python3.7m/lapsolver
         cwd: /private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/
    Complete output (79 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.9-x86_64-3.7
    creating build/lib.macosx-10.9-x86_64-3.7/lapsolver
    copying lapsolver/__init__.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver
    creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
    copying lapsolver/tests/__init__.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
    copying lapsolver/tests/test_files.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
    copying lapsolver/tests/test_dense.py -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/tests
    running egg_info
    writing lapsolver.egg-info/PKG-INFO
    writing dependency_links to lapsolver.egg-info/dependency_links.txt
    writing top-level names to lapsolver.egg-info/top_level.txt
    reading manifest file 'lapsolver.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'lapsolver.egg-info/SOURCES.txt'
    creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/data
    creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs0.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs1.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs10.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs2.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs3.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs4.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs5.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs6.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs7.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs8.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    copying lapsolver/data/dense/costs9.npz -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/data/dense
    creating build/lib.macosx-10.9-x86_64-3.7/lapsolver/etc
    copying lapsolver/etc/benchmark-dtype-int.png -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/etc
    copying lapsolver/etc/benchmark-dtype-numpy.float32.png -> build/lib.macosx-10.9-x86_64-3.7/lapsolver/etc
    running build_ext
    Traceback (most recent call last):
      File "/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py", line 22, in run
        out = subprocess.check_output(['cmake', '--version'])
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 411, in check_output
        **kwargs).stdout
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 488, in run
        with Popen(*popenargs, **kwargs) as process:
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 800, in __init__
        restore_signals, start_new_session)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/subprocess.py", line 1551, in _execute_child
        raise child_exception_type(errno_num, err_msg, err_filename)
    FileNotFoundError: [Errno 2] No such file or directory: 'cmake': 'cmake'
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py", line 88, in <module>
        keywords='hungarian munkres kuhn linear-sum-assignment bipartite-graph lap'
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages/setuptools/__init__.py", line 144, in setup
        return distutils.core.setup(**attrs)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/command/install.py", line 545, in run
        self.run_command('build')
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/Users/mwmathis/opt/anaconda3/envs/DLC-anipose/lib/python3.7/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py", line 25, in run
        ", ".join(e.name for e in self.extensions))
    RuntimeError: CMake must be installed to build the following extensions: lapsolverc
    ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py'"'"'; __file__='"'"'/private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-install-_qt4q6_o/lapsolver/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/ct/4w_vk2wn10xb_xj2fm7d6vqc0000gn/T/pip-record-ppsvuzrf/install-record.txt --single-version-externally-managed --compile --install-headers /Users/mwmathis/opt/anaconda3/envs/DLC-anipose/include/python3.7m/lapsolver Check the logs for full command output.
(DLC-anipose) mwmathis@Mackenzies-MacBook-Pro Desktop % 
```

I also noticed that there is a bit of an issue with the 3 packages, such that order of operation matters a bit, which is suboptimal: can anipose be pinged to `'opencv-python~=3.4'`?
DLC setup requires: `'opencv-python~=3.4' `
calligator requires: `'opencv-python~=3.4', 'opencv-contrib-python~=3.4' `
anipose just has: `'opencv-python'`

**Comment by lambdaloop** [comment]
Hi @MMathisLab ,

Thank you for this issue, it is very helpful! 

In fact, the `lapsolver` dependency is only used in a deprecated "clusters" 2D filter, which I found to not actually improve the 2D tracking. I've now removed this filter and the `lapsolver` dependency.

That's a good catch about the `opencv-python~=3.4`. I've now setup Anipose to require that version as well. 

I've updated Anipose to version 0.6.7 on pip which fixes these issues, let me know if it works!
